### PR TITLE
Update texts on Step 1 depending on the platform

### PR DESCRIPTION
**Pull request by prupipho**
Issue: #1672

* Modify README.md in the templates to make it clear that we are still working on this functionality
* Add different messages files per platform to display on Step 1

image

### CODE REVIEW

**Comment by prupipho** [comment]
Rename Page components PR #1698  should go first

File: src/client/src/pages/NewProjectPage/index.tsx [hunk 1]
```diff
@@ -2,13 +2,24 @@ import * as React from "react";
...
 
 import styles from "./styles.module.css";
-import messages from "./messages";
+import messagesWeb from "./messagesWeb";
+import messagesReactNative from "./messagesReactNative";
+import { PLATFORM } from "../../utils/constants/constants";
 
 type Props = InjectedIntlProps;
 
 const NewProject = (props: Props) => {
```
Comment (prupipho) [inline src/client/src/pages/NewProjectPage/index.tsx:15]: Try passing this from props.

File: src/client/src/pages/NewProjectPage/index.tsx [hunk 2]
```diff
@@ -2,13 +2,24 @@ import * as React from "react";
...
+  const platform: string = useSelector((state: AppState) => state.config.platform);
+  let messages = messagesWeb;
+  switch(platform){
+    case PLATFORM.REACTNATIVE:
+      messages = messagesReactNative;
+      break;
+  }
```
Comment (prupipho) [inline src/client/src/pages/NewProjectPage/index.tsx]:
```suggestion
  const messages = (platform == PLATFORM.WEB) ? messagesWeb : messagesReactNative;
```

File: src/client/src/pages/NewProjectPage/messagesReactNative.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  header: {
+    id: "newProjectPage.reactNative.header",
+    defaultMessage: "Create Your React Native App in Seconds",
+  },
+  body: {
+    id: "newProjectPage.reactNative.body",
+    defaultMessage: `* * This is still work in progress. * *
+    We will appreciate any feedback provided.
+    * * * *
+    Give your react native project a name, choose where to create it, then click 'Next' to get started.`,
```
Comment (prupipho) [inline src/client/src/pages/NewProjectPage/messagesReactNative.ts:13]:
```suggestion
    defaultMessage: "Give your react native project a name, choose where to create it, then click 'Next' to get started.",
```

File: src/client/src/pages/NewProjectPage/messagesReactNative.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  },
```
Comment (prupipho) [inline src/client/src/pages/NewProjectPage/messagesReactNative.ts]:
```suggestion
  header: {
    id: "newProjectPage.reactNative.header",
    defaultMessage: "Create Your React Native App in Seconds (WIP)",
  },
  body: {
    id: "newProjectPage.reactNative.body",
    defaultMessage: "Give your react native project a name, choose where to create it, then click 'Next' to get started.",
  },
```

File: templates/RN/Projects/AddInitialProjectStructure/README.md [hunk 5]
```diff
@@ -1 +1,9 @@
-﻿*** REACT NATIVE *** project wts.projectName created
\ No newline at end of file
+﻿# WORK IN PROGRESS
+This is still work in progress.
+We will appreciate any feedback provided.
+- Opinion about the mockups available in [repo]().
```
Comment (prupipho) [inline templates/RN/Projects/AddInitialProjectStructure/README.md:4]: We need to add the Mock-ReactNative repo here once we merge it.

File: src/client/src/pages/NewProjectPage/messagesReactNative.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { defineMessages } from "react-intl";
+
+const messagesReactNative = defineMessages({
+  header: {
+    id: "newProjectPage.reactNative.header",
+    defaultMessage: "Create Your React Native App in Seconds (WIP)",
+  },
+  body: {
+    id: "newProjectPage.reactNative.body",
+    defaultMessage: "Give your react native project a name, choose where to create it, then click 'Next' to get started.",
```
Comment (sibille) [inline src/client/src/pages/NewProjectPage/messagesReactNative.ts]: We shiould correct "React Native" casing or remove it from the body text.
```suggestion
    defaultMessage: "Give your project a name, choose where to create it, then click 'Next' to get started.",
```

**Review by sibille — DISMISSED** [review]

**Review by sibille — APPROVED** [review]

**Review by dgomezc — APPROVED** [review]

**Pull request merged**